.github/workflows/node-build.yml

```diff
@@ -338,7 +338,7 @@ jobs:
       - name: Scan docker image
         run: |
           docker images
-          /tmp/trivy image --ignore-unfixed --format table --vuln-type os,library --exit-code 1 --severity HIGH --input /tmp/${{ github.sha }}-${{ matrix.package }}-${{ matrix.platform.name }}-${{ needs.version-generator.outputs.version }}.tar
+          /tmp/trivy image --db-repository ghcr.io/aquasecurity/trivy-db,public.ecr.aws/aquasecurity/trivy-db --java-db-repository ghcr.io/aquasecurity/trivy-java-db,public.ecr.aws/aquasecurity/trivy-java-db --ignore-unfixed --format table --vuln-type os,library --exit-code 1 --severity HIGH --input /tmp/${{ github.sha }}-${{ matrix.package }}-${{ matrix.platform.name }}-${{ needs.version-generator.outputs.version }}.tar
```

Comment:
Trivy scan kept failing because of 
- https://github.com/aquasecurity/trivy/discussions/7668

added fix from [here](https://github.com/aquasecurity/trivy/discussions/7668#discussioncomment-10884984) until https://github.com/aquasecurity/trivy/pull/7679 is merged